.circleci/config.yml

```diff
@@ -26,13 +26,16 @@ jobs:
   snyk-scan:
     docker:
       - image: cimg/python:3.10
+      - image: cimg/openjdk:17.0.6
     steps:
       - attach_workspace:
           at: ~/project
+      - run: rm -rf mvnw mvnw.cmd .mvn/wrapper
       - general-platform-helpers/step-load-dependencies
       - general-platform-helpers/step-run-snyk-monitor:
           scan-all-projects: true
           skip-unresolved: false
+          run-on-non-main: true
```

Comment:
Make sure to remove this after this testing is done.

Reply:
Removing this would mean all PRs running on feature branches are bound to fail because the job `snyk-scan` will error out. cc: @bretterer

Reply:
@aravindanranganathan-okta hmmm, you're correct. Apologies,  this shouldn't fail if set to false or not be present. It should simply skip the scan - I'll chase this down.  Feel free to leave it.